frontend/src/component/photo/tile.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <v-flex
+      :data-uid="photo.UID"
+      v-bind:class="{ selected: isSelected }"
+      class="p-photo"
+      xs4 sm3 md2 lg1 d-flex
+  >
+    <v-hover>
+      <v-card tile slot-scope="{ hover }"
+              @contextmenu="onContextMenu($event, index)"
+              :class="isSelected ? 'elevation-10 ma-0' : 'elevation-0 ma-1'"
+              :title="photo.Title">
+        <v-img :src="photo.thumbnailUrl('tile_224')"
+               aspect-ratio="1"
+               class="accent lighten-2 clickable"
+               @mousedown="onMouseDown($event, index)"
+               @click.stop.prevent="onClick($event, index)"
+        >
+          <v-layout
+              slot="placeholder"
+              fill-height
+              align-center
+              justify-center
+              ma-0
+          >
+            <v-progress-circular indeterminate
+                                 color="accent lighten-5"></v-progress-circular>
+          </v-layout>
+
+          <v-layout
+              fill-height
+              align-center
+              justify-center
+              ma-0
+              class="p-photo-live"
+              style="overflow: hidden;"
+              v-if="photo.Type === 'live'"
+              v-show="hover"
+          >
+            <video width="224" height="224" autoplay loop muted playsinline :key="photo.videoUrl()">
+              <source :src="photo.videoUrl()" type="video/mp4">
+            </video>
+          </v-layout>
+
+          <v-btn v-if="hidePrivate && photo.Private" :ripple="false"
+                 icon flat small absolute
+                 class="p-photo-private opacity-75">
+            <v-icon color="white">lock</v-icon>
+          </v-btn>
+
+          <v-btn v-if="hover || isSelected" :ripple="false"
+                 icon flat small absolute
+                 :class="isSelected ? 'p-photo-select' : 'p-photo-select opacity-50'"
+                 @click.stop.prevent="onSelect($event, index)">
+            <v-icon v-if="isSelected" color="red"
```

Comment:
I changed the color to `red` to check that my changes were reflected in the UI. I have to changed it back to `white` before this PR gets merged.